http-gateway/web: Allow setting the visibility for the snippet service

jkralik opened the pull request:
**Snippet Service Menu Visibility**: The visibility of the snippet service menu is now controlled through the well-known configuration settings. This change ensures that the menu visibility can be dynamically managed and easily adjusted based on configuration.

**Brave/Safari Browser Device Code Retrieval**: Updated the approach for obtaining the device code to ensure compatibility with the Brave/Safari browser. This modification addresses specific challenges related to Brave's unique privacy features and enhances the user experience for Brave/Safari browser users.

jkralik committed:
http-gateway/web: Allow setting the visibility for the snippet service

Danielius1922 commented on .github/workflows/test.yml:
```diff
@@ -144,6 +144,55 @@ jobs:
+      - name: Set up Helm
+        uses: azure/setup-helm@v4.2.0
+        with:
+          version: v3.14.4
```
is this the version we have installed or what's the significance of v3.14.4?

jkralik replied:
removed

jkralik committed:
helm/mongodb: fix for secret with empty file

jkralik committed:
fix scylla and comment

PatrikMatiasko committed:
snippet service menu visibility + brave browser test (#1345)

Snippet Service Menu Visibility: The visibility of the snippet service menu is now controlled through the well-known configuration settings. This change ensures that the menu visibility can be dynamically managed and easily adjusted based on configuration.

Brave/Safari Browser Device Code Retrieval: Updated the approach for obtaining the device code to ensure compatibility with the Brave/Safari browser. This modification addresses specific challenges related to Brave's unique privacy features and enhances the user experience for Brave browser users.

---------

Co-authored-by: Jozef Kralik <jozef.kralik@plgd.dev>
Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>
